filetracker/scripts/migrate_test.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for migrate script."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from multiprocessing import Process
+import os
+import shutil
+import tempfile
+import time
+import unittest
+
+from filetracker.client import Client, FiletrackerError
+from filetracker.scripts import migrate
+from filetracker.servers.run import main as server_main
+
+_TEST_PORT_NUMBER = 45785
+
+
+class MigrateScriptTest(unittest.TestCase):
```

Comment:
If we've got tests for the migration script, maybe we could add a test that tries to upload a file with forbidden name and files to see if other files are migrated correctly in such a scenario?